NewEssentials/translations.yaml

```diff
@@ -134,4 +134,7 @@ god:
   god: "You are now a God!"
   god_other: "{Player} is now a God!"
   mortal: "You are now a mortal!"
-  mortal_other: "{Player} has reappeared!"
\ No newline at end of file
+  mortal_other: "{Player} has reappeared!"
+  
+tphere:
+  successfull_tp: "{Player} has been tped to you"
```

Comment:
Remove the l from successful (sorry I know this is petty but better than changing it later)

Reply:
I will address this